pkgs/tools/misc/owofetch/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "owofetch";
+  version = "0.3.1";
+
+  src = fetchFromGitHub {
+    owner = "netthier";
+    repo = "owofetch-rs";
+    rev = "v${version}";
+    sha256 = "sha256-I8mzOUvm72KLLBumpgn9gNyx9FKvUrB4ze1iM1+OA18=";
+  };
+
+  cargoSha256 = "sha256-rfN4QERs1H1G7ZZim//78vlxbYfU4Cx7SYYUz/QLKeU=";
+
+  meta = with lib; {
+    description = "Alternative to *fetch, uwuifies all stats";
+    homepage = "https://github.com/netthier/owofetch-rs";
+    license = licenses.gpl3;
```

Comment:
If possible we should disambiguate between GPL-3 only and GPL-3 or later.

https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/unclear-gpl.md

Reply:
As far as I can tell, this is a `gpl3Only` project. I'm not familiar at all with the GPL licenses, could you confirm if that is the case? I'll push a fix after that.

Reply:
You are right: there is no indication either way. `only` is the more prudent interpretation then.

Reply:
Cool, I'll push that now.